_posts/2018-08-08-survey.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Launching the 2018 State of Rust Survey"
+author: The Rust Community Team
+description: "Hearing from you about the third year of Rust"
+---
+
+It's that time again! Time for us to take a look at how the Rust project is doing, and where we should plan for the future. The Rust Community Team is pleased to announce our [2018 State of Rust Survey][survey]! Whether or not you use Rust today, we want to know your opinions. Your responses will help the project understand its strengths and weaknesses, and to establish development priorities for the future.
+
+Completing this survey should take about 10 to 15 minutes, and is anonymous unless you choose to give us your contact information. We will be accepting submissions until September 8th, and we will write up our findings a month or so afterwards to [blog.rust-lang.org]. You can see last year’s results [here][2017 survey].
+
+This year, volunteers have also translated the survey into many languages! You can now take the survey in a number of languages:
```

Comment:
"many languages" and "number of languages" sounds repetitive, what about "You can now take the survey in:"?